aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -1233,7 +1235,7 @@ pub fn stake_update_network_and_fullnode_addresses(
 }
 
 /// Withdraw from `account`'s inactive stake.
-pub fn stake_withdraw() -> TransactionPayload {
+pub fn stake_withdraw(withdraw_amount: u64) -> TransactionPayload {
```

Comment:
are these changes required for the table?

Reply:
no, rebase issue (someone force pushed main? seeing it elsewhere as well.)